casanovo/denovo/dataloaders.py

```
@@ -240,6 +253,44 @@ def _make_dataset(
 
         return dataset
 
+    def _train_dia_align(self, annotation_paths):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_train_dia_align`

Add return type annotation: `None`

(ANN202)